glossary.yaml

```diff
@@ -99,7 +99,7 @@ terms:
     name: "Private Network"
   - definition: "A blockchain application is a software application that has integrated a blockchain in its architecture as the storage layer for part of, or, all of its data. This includes software applications that do not own their own contract on the blockchain and instead only interact with the system contracts of the blockchain."
     name: "Blockchain Application"
-  - definition: "A block is considered irreversible (i.e. immutable) on an EOSIO based blockchain when 2/3rd of the currently elected block producers have acknowledged it."
+  - definition: "Irreversible blocks are blocks that contain confirmed, final transactions. A block is considered irreversible (i.e., immutable) on an EOSIO-based blockchain when a supermajority, consisting of 2/3rds, plus one of the currently elected block producers have confirmed the block."
```

Comment:
the supermajority is 2/3rds plus one, not 2/3rds.
supermajority, consisting of 2/3rds, plus one of --> supermajority, consisting of 2/3rds plus one, of

Reply:
Clear now?